tests/test_logical_types.py

```diff
@@ -76,20 +78,25 @@ def test_logical_types():
             == data2['time-millis'].microsecond)
 
 
-def test_not_logical_ints():
-    data1 = {
-        'date': 1,
-        'timestamp-millis': 2,
-        'timestamp-micros': 3,
-        'uuid': uuid4(),
-        'time-millis': 4,
-        'time-micros': 5,
+if os.name == 'nt' and sys.version_info[:2] == (3, 6):
+    @pytest.mark.skip(reason='Python Bug: https://bugs.python.org/issue29097')
```

Comment:
You may want to look into using `skipif`.

https://docs.pytest.org/en/latest/skipping.html